os-config.tpl.yml

```diff
@@ -146,6 +146,7 @@ rancher:
       - /usr/bin/ros:/usr/bin/cloud-init-save:ro
       - /usr/bin/ros:/usr/sbin/netconf:ro
       - /usr/bin/ros:/usr/sbin/wait-for-docker:ro
+      - /usr/rancher:/usr/share/rancher:ro
```

Comment:
I think this might make more sense under `system-volumes`. This will also cause upgrade issues for persistent consoles...